salt/backups/templates/restore_mongodb.sh

```diff
@@ -15,5 +15,5 @@ PASSPHRASE={{ settings.duplicity_passphrase }} /usr/bin/duplicity restore\
                       --port {{ settings.get('port', 27017) }} \
                       --password {{ settings.password }} --username admin \
                       --authenticationDatabase admin \
-                      --db {{ settings.database }} \
-                      --drop {{ backupdir }}/{{ settings.database }}
+                      --db {{ settings.restore_to }} \
+                      --drop {{ backupdir }}/{{ settings.restore_from }}
```

Comment:
For mongo this won't quite work because all of the databases will be downloaded from S3 at once, so it would be best to  add a loop in the script to run the mapping all at once.